docs/offlineinstallation/offline-sc4snmp.md

```diff
@@ -0,0 +1,156 @@
+# Offline SC4SNMP installation
+
+## Local machine with internet access
+To install SC4SNMP offline first some packages must be downloaded from github release and then moved
+to the sc4snmp installation server. Those packages are:
+
+- `dependencies-images.tar`
+- `splunk-connect-for-snmp-chart.tar`
+
+Moreover, SC4SNMP Docker image must be pulled, saved as `.tar` package and then moved to the server as well. 
+This process requires Docker installed locally.
+
+Image can be pulled from the following repository: `ghcr.io/splunk/splunk-connect-for-snmp/container:<tag>`.
+
+Example of docker pull command:
+
+```bash
+docker pull ghcr.io/splunk/splunk-connect-for-snmp/container:1.6.2
```

Comment:
Maybe we should change`1.6.2` to tag variable and desribe from where to get current newest release?